C-Plus-Plus/README.md

```diff
@@ -522,3 +522,4 @@ _add list here_
 - [Activity Selection](other/ActivitySelection.cpp)
 - [Job sheduling with given deadlines](other/job_sheduling_with_deadlines.cpp)
 - [Union and Intersection of arrays](other/Union_and_Intersection_of_arrays.cpp)
+- [Find repeating and missing number in array](other/repeated_missing.cpp)
```

Comment:
Rewrite this ` [Find repeating and missing number in array]` to like that `[Repeating and Missing Number]`